### PR TITLE
chore : adding s3bucket_url to prod global manifest PROD

### PR DESCRIPTION
**Pull request by atharvar28**
adding s3BucketURL to the global manifest for the purpose to run the covid19-notebook-etl job

### CODE REVIEW

File: chicagoland.pandemicresponsecommons.org/manifest.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "tier_access_level": "regular",
     "tier_access_limit": "5",
     "useryaml_s3path": "s3://cdis-gen3-users/covid19/user.yaml",
+    "covid19_etl_bucket": "https://opendata.datacommons.io/",
```
Comment (jawadqur) [inline chicagoland.pandemicresponsecommons.org/manifest.json:50]: I don't think this is correct. the current value is this: 
```            - name: S3_BUCKET
              value: s3://opencdn-databucket-gen3
```

@paulineribeyre Can you confirm the correct bucket name here?

Comment (paulineribeyre) [reply chicagoland.pandemicresponsecommons.org/manifest.json:50]: ah shoot, yeah, the bucket name is `s3://opencdn-databucket-gen3` and the CDN URL is `https://opendata.datacommons.io`

@atharvar28, I forgot the portal uses the URL, not the bucket name, so we won't be able to pull that value from the manifest for the portal. But that's ok, the portal can keep pulling from its own config, and the ETLs can pull from the manifest

Comment (atharvar28) [reply chicagoland.pandemicresponsecommons.org/manifest.json]: I have updated the s3 url. Thank you guys 👍

**Review by jawadqur — APPROVED** [review]

**Review by jawadqur — APPROVED** [review]

**Review by paulineribeyre — COMMENTED** [review]
lgtm but let's get it to work in QA before merging changes to prod :-)

**Review by jawadqur — APPROVED** [review]
Looks good to me too, assuming it's working in QA let's push to prod 🙌

**Pull request merged**